src/web/methods/checkers.methods.ts

```diff
@@ -8,7 +8,7 @@ import variableStore from '../../core/variable-store.helper';
 
 const valueToTextTransformer = createValueToTextTransformer();
 
-export const checkNumberOfElements = (currentPage: BasePage, numberExpression: string, elementName: string) => {
+export const checkNumberOfElements = (currentPage: BasePage, numberExpression: string, elementName: any) => {
```

Comment:
You union as a type, probably string | protractor.ElementArrayFinder

Reply:
Done